Tools/CMake/AMReXSYCL.cmake

```diff
@@ -68,6 +68,16 @@ if (AMReX_MPI)
     $<${_cxx_dpcpp}:-fsycl-unnamed-lambda>)
 endif ()
 
+if(AMReX_DPCPP_ONEDPL)
+    # TBB and PSTL are broken in oneAPI 2021.3.0
+    # https://software.intel.com/content/www/us/en/develop/articles/intel-oneapi-dpcpp-library-release-notes.html#inpage-nav-2-3
+    if(DPCPP_VERSION VERSION_GREATER_EQUAL 2021.3.0 AND DPCPP_VERSION VERSION_LESS 2021.4.0)
```

Comment:
The issue has existed at least since 2021.1.1, and probably won't be fixed until glibc version 10 is gone.  So we probably should drop the version test.

Reply:
I tested this PR with 2021.2.  It fails.  But if I remove the test, it compiles.

Reply:
Thanks for the test, will update!